symengine/tests/eval/test_eval_double.cpp

```diff
@@ -81,6 +82,11 @@ TEST_CASE("eval_double: eval_double", "[eval_double]")
         { mul(EulerGamma, integer(10)), 5.7721566490153286 },
         { max({r2, r1}), 0.841470984808},
         { min({add(r1, r4), r2}), 0.479425538604},
+        { gamma(integer(2)), 1 },
```

Comment:
Then you should add a test that actually evaluates a `Gamma` instance. Here you are evaluating a `Integer` object